nitro-protocol/contracts/interfaces/IForceMove.sol

```diff
@@ -2,97 +2,45 @@
 pragma solidity 0.7.6;
 pragma experimental ABIEncoderV2;
 
-import './IForceMoveApp.sol';
+import './INitroTypes.sol';
 
 /**
  * @dev The IForceMove interface defines the interface that an implementation of ForceMove should implement. ForceMove protocol allows state channels to be adjudicated and finalized.
  */
-interface IForceMove {
-    struct Signature {
-        uint8 v;
-        bytes32 r;
-        bytes32 s;
-    }
-
-    struct FixedPart {
-        uint256 chainId;
-        address[] participants;
-        uint48 channelNonce;
-        address appDefinition;
-        uint48 challengeDuration;
-    }
-
-    struct State {
-        // participants sign the hash of this
-        bytes32 channelId; // keccack(chainId,participants,channelNonce,appDefinition,challengeDuration)
-        bytes appData;
-        bytes outcome;
-        uint48 turnNum;
-        bool isFinal;
-    }
-
+interface IForceMove is INitroTypes {
     /**
      * @notice Registers a challenge against a state channel. A challenge will either prompt another participant into clearing the challenge (via one of the other methods), or cause the channel to finalize at a specific time.
      * @dev Registers a challenge against a state channel. A challenge will either prompt another participant into clearing the challenge (via one of the other methods), or cause the channel to finalize at a specific time.
      * @param fixedPart Data describing properties of the state channel that do not change with state updates.
-     * @param variableParts An ordered array of structs, each decribing the properties of the state channel that may change with each state update. Length is from 1 to the number of participants (inclusive).
-     * @param sigs An array of signatures that support the state with the `largestTurnNum`. There must be one for each participant, e.g.: [sig-from-p0, sig-from-p1, ...]
-     * @param whoSignedWhat An array denoting which participant has signed which state: `participant[i]` signed the state with index `whoSignedWhat[i]`.
+     * @param signedVariableParts An ordered array of structs, that can be signed by any number of participants, each struct decribing the properties of the state channel that may change with each state update. Length is from 1 to the number of participants (inclusive).
```

Comment:
```suggestion
     * @param signedVariableParts An ordered array of structs, that can be signed by any number of participants, each struct describing the properties of the state channel that may change with each state update.
```
I don't think we need to say this (and I'm not 💯 % sure it is true?)

Reply:
Yeah, I was not sure either. I think, it is better to specify all parameter restrictions in one place in docs, but not in every function.